Fail e2e tests on travis.

```
[1] INFO Selenium process finished.
[1] nightwatch --config test/nightwatch.conf.js --env ie9,ie10,ie11,ie_edge,chrome,firefox,safari exited with code 0
--> Sending SIGTERM to other processes..
[0] node test/e2e/static-server.js exited with code null
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! summernote@0.8.9 test:e2e-saucelabs: `npx bower install && npm run dist && npm run download:selenium && concurrently --kill-others 'node test/e2e/static-server.js' 'nightwatch --config test/nightwatch.conf.js --env ie9,ie10,ie11,ie_edge,chrome,firefox,safari'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the summernote@0.8.9 test:e2e-saucelabs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2017-10-15T07_46_01_537Z-debug.log
The command "if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then npm run test:e2e-saucelabs; fi" exited with 1.
cache.2
store build cache
0.01s
5.55snothing changed, not updating cache
sauce_connect.stop
Stopping Sauce Connect
```

[It](https://travis-ci.org/summernote/summernote/builds/288133686) looks like exit code problem.

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,10 +11,12 @@ before_install:
 before_script:
 - node --version
 - npm --version
+- if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then node test/e2e/static-server.js & fi
 script:
 - npm run test-travis
 - if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then npm run test-saucelabs; fi
 - if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then npm run test:e2e-saucelabs; fi
+- if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then pkill node; fi
 before_deploy:
 - npm install
 - npm run build

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "test": "grunt test",
     "download:selenium": "if [ ! -e test/libs/selenium-server-standalone.jar ]; then wget http://selenium-release.storage.googleapis.com/3.6/selenium-server-standalone-3.6.0.jar -O test/libs/selenium-server-standalone.jar; fi",
     "test:e2e": "npm run download:selenium && concurrently --kill-others 'node test/e2e/static-server.js' 'nightwatch --config test/nightwatch.conf.js'",
-    "test:e2e-saucelabs": "npx bower install && npm run dist && npm run download:selenium && concurrently --kill-others 'node test/e2e/static-server.js' 'nightwatch --config test/nightwatch.conf.js --env ie9,ie10,ie11,ie_edge,chrome,firefox,safari'",
+    "test:e2e-saucelabs": "npx bower install && npm run dist && npm run download:selenium && nightwatch --config test/nightwatch.conf.js --env ie9,ie10,ie11,ie_edge,chrome,firefox,safari",
     "build": "grunt build",
     "dist": "grunt dist",
     "test-travis": "grunt test-travis --verbose",

--- a/test/e2e/static-server.js
+++ b/test/e2e/static-server.js
@@ -11,9 +11,10 @@ var server = http.createServer(function onRequest(req, res) {
 
 // Listen
 server.listen(3000, function() {
-  console.log('Launched static-server on 3000 port.');
+  console.log('Launched static-server on 3000 port');
 });
 
 process.on('SIGTERM', function() {
+  console.log('static-server terminated');
   process.exit(0);
 });

